06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -139,5 +143,14 @@ describe('Inverted index create method', () => {
     it('should return true for correctFileContent', () => {
       expect(correctFileContent).toBeTruthy();
     });
+
+    it('should return false for pakage.json as book', () => {
+      console.log(packageJsonBook);
+      expect(packageJsonBook).toEqual(false);
+    });
+
+    it('should return false wrong file content', () => {
```

Comment:
Is this suppose to test for an invalid file? Your specification should communicate that please